proto/motor/v1/service.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+// Package Motor is used for defining a Motor node and its properties.
+package sonrio.motor.api.v1;
+
+import "motor/v1/request.proto";
+import "motor/v1/response.proto";
+
+option go_package = "github.com/sonr-io/sonr/third_party/types/motor/api/v1";
+
+service MotorService {
```

Comment:
i think we should remove this new rpc service interface for now. code could be moved to a new branch to revisit.